README.md

```diff
@@ -56,47 +56,41 @@ two containers, one on each host.
 On `$HOST1` run
 
     host1$ weave launch
-    host1$ C=$(weave run 10.2.1.1/24 -t -i ubuntu)
+    host1$ weave launch-dns
+    host1$ weave run --name h1c1 -t -i ubuntu
 
-The first line starts the weave router, in a container. This needs to
-be done once on each host. The required docker image for the weave
-router container is downloaded automatically. There is also a `weave
-setup` command for downloading this and other images required for
+The first two lines start the weave router and DNS, each inside their
+own container - this needs to be done once per host. Whilst the
+required docker images are downloaded automatically, there is also a
+`weave setup` command for downloading all the images required for
```

Comment:
`weave setup` could go elsewhere; it is not a key feature to learn about on first contact.